$NetBSD: patch-pyproject.toml,v 1.3 2026/01/09 05:53:39 adam Exp $

pkgsrc only has Python 3.10+, so the dependency is not needed.
(It introduces a dependency cycle.)

--- pyproject.toml.orig	2026-01-08 16:33:42.000000000 +0000
+++ pyproject.toml
@@ -37,7 +37,6 @@ dependencies = [
   "pyproject_hooks",
   # not actually a runtime dependency, only supplied as there is not "recommended dependency" support
   'colorama; os_name == "nt"',
-  'importlib-metadata >= 4.6; python_full_version < "3.10.2"',  # Not required but fixes a stdlib bug
   'tomli >= 1.1.0; python_version < "3.11"',
 ]
 
